examples/04_react_router/src/App.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  unstable_HistoryRouter as BrowserRouter,
```

Comment:
I wonder why you call it BrowserRouter instead of HistoryRouter.

Reply:
Leftover from porting my own example. Fixed in [480239b](https://github.com/jotaijs/jotai-location/pull/25/commits/480239b9f37fb0f821356c4a4c40971234c5d149)! 👌